docs/ides/fleet.md

```diff
@@ -0,0 +1,26 @@
+# JetBrains Fleet
+
+JetBrainsFleet comes with essential features out of the box, allowing you to be
+productive immediately. Just one click transforms Fleet from a lightweight
+editor into a powerful IDE.
```

Comment:
Is this their tagline? Do we believe it to be true?

Reply:
It's from their page. Fleet is still a public preview so can't say anything about it now. I will remove it.

Reply:
Done